script/main.py

```diff
@@ -21,6 +22,30 @@ def init_argparser():
   return args
 
 
+def find_record_with_dob(people_json):
+  record_names = []
+  dict_json_to_list = people_json['results']
+  for dict_single_record in dict_json_to_list:
+    if 'dob' in dict_single_record:
+      record_names.append(dict_single_record)
+  return record_names
+
+
+def create_new_record_with_dob_in_json(records):
+  for record in records:
+    days_left = 0
+    born = record['dob']['date']
+    date_born_dt = datetime.strptime(born, '%Y-%m-%dT%H:%M:%S.%fZ')
```

Comment:
This can be improved.
Check for example this link how to gen correct date difference:
https://stackoverflow.com/questions/8419564/difference-between-two-dates-in-python
or check official documentation. There is an example similar to this task:
https://docs.python.org/3/library/datetime.html#examples-of-usage-date

Reply:
This  '%Y-%m-%dT%H:%M:%S.%fZ' is format of the date in the json file.
I don't know how I can use "data", when trying to make it simpler I get an error:
`ValueError("unconverted data remains: %s" % ValueError: unconverted data remains: T11:50:25.558Z`